### PR TITLE
Add more vehicle tabs to the gui

### PR DESCRIPTION
**Pull request by killerswin2**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    Adds more tabs. 

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:
Screenshot 2023-04-04 172718
Screenshot 2023-04-04 172737
Screenshot 2023-04-04 172744
Screenshot 2023-04-04 172754

### CODE REVIEW

File: A3A/addons/gui/dialogues/buyVehicleDialog.hpp [hunk 1]
```diff
@@ -50,16 +72,52 @@ class A3A_BuyVehicleDialog : A3A_TabbedDialog
         };
 
         // Main content
-        class VehicleTab : A3A_DefaultControlsGroup
+        class civilainVehicleTab : A3A_DefaultControlsGroup
```
Comment (jaj22) [inline A3A/addons/gui/dialogues/buyVehicleDialog.hpp:75]: Doesn't matter, but typo in classname.

File: A3A/addons/gui/dialogues/buyVehicleDialog.hpp [hunk 2]
```diff
@@ -25,23 +25,45 @@ class A3A_BuyVehicleDialog : A3A_TabbedDialog
...
+                    onButtonClick = "[""switchTab"", [""civilian""]] call A3A_fnc_buyVehicleDialog";
                     x = 0;
                     y = 0;
                     w = 30 * GRID_W;
                     h = 5 * GRID_H;
                 };
 
+                 class rebelVehicleTabButton : A3A_Button
+                {
+                    idc = A3A_IDC_VEHICLETABBUTTON;
```
Comment (jaj22) [inline A3A/addons/gui/dialogues/buyVehicleDialog.hpp:41]: Kinda funny that this works, but I guess it should really have a different idc, or -1 if it's not used? Ditto for the statics tab.

**Review by Bob-Murphy — APPROVED** [review]
lgtm but yes, the typo in the class should be fixed

**Review by jaj22 — APPROVED** [review]
Good to go.

**Pull request merged**